x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/prebuilt_rules/delete_all_prebuilt_rule_assets.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Client } from '@elastic/elasticsearch';
+import { SECURITY_SOLUTION_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-server';
+
+/**
+ * Remove all prebuilt rule assets from the security solution savedObjects index
+ * @param es The ElasticSearch handle
+ */
+export const deleteAllPrebuiltRuleAssets = async (es: Client): Promise<void> => {
```

Comment:
I explicitly duplicated the helper functions even if they are still used in `detection_engine_api_integration` because I see this as a temporary intermediate state, and in the short term all tests and their utils will be migrated. And `detection_engine_api_integration` will be fully deleted.
Do you see it another way?

Reply:
> in the short term all tests and their utils will be migrated

@jpdjere We don't know how much time it's going to take exactly, that's why I'm looking for ways to reduce code duplication. I'm ok to keep it as is in this PR, though.